tasks/configure.yml

```diff
@@ -78,8 +83,26 @@
 - name: Flush handlers to ensure Zookeeper is up to date
   meta: flush_handlers
 
-- name: Wait for Zookeeper port
+- name: Wait for Zookeeper client port
   wait_for:
     port: "{{ zookeeper.client_port }}"
     state: started
-    timeout: 30
+    timeout: 60
+
+- name: Copy zookeeper check script
+  become: yes
+  become_user: "{{ zookeeper.user }}"
+  copy:
+    src: zookeeper_check
+    dest: "/home/{{ zookeeper.user }}/bin/zookeeper_check"
+    owner: "{{ zookeeper.user }}"
+    group: "{{ zookeeper.group }}"
+    mode: 0750
+
+- name: Run zookeeper check script
+  become: yes
+  become_user: "{{ zookeeper.user }}"
+  shell: "/home/{{ zookeeper.user }}/bin/zookeeper_check"
+  async: 7200
+  poll: 0
```

Comment:
> Alternatively, if you do not need to wait on the task to complete, you may “fire and forget” by specifying a poll value of 0

Ansible playbook run is not waiting for this task to complete.

If you want to wait for the `zookeeper_check` result, just set it in a range of 10-60.